tests/redis-vector.test.ts

```
@@ -0,0 +1,182 @@
+  hset: mock(async () => 'OK'),
+  expire: mock(async () => 1),
+  del: mock(async () => 1),
+  disconnect: mock(() => {}),
+  on: mock(() => {}),
+};
+
+mock.module('ioredis', () => {
+  return {
+    default: class Redis {
+      constructor() {
+        return mockRedisClient;
```

Biome (2.3.14)

[error] The constructor should not return a value.

The constructor is here:

Returning a value from a constructor may confuse users of the class.

(lint/correctness/noConstructorReturn)